packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +293,50 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   * Add subcommand and sub-sub commands for 'setup' subcommand.
+   * @param {import('argparse').SubParser} subParser
+   */
+  static _addSetupToParser(subParser) {
+    const setupParser = subParser.add_parser('setup', {
+      add_help: true,
+      help: `Install latest official drivers and/or plugins for specific usage ` +
+        `if APPIUM_HOME has no drivers and plugins. Default is 'mobile' set.`,
+    });
+
+    ArgParser._patchExit(setupParser);
+    const extSubParsers = setupParser.add_subparsers({
+      dest: `setupCommand`,
+    });
+
+    const parserSpecs = [
+      {
+        command: SUBCOMMAND_MOBILE,
+        help: `Install latest drivers for Android and iOS '${_.join(MOBILE_DRIVERS, ',')}' ` +
```

Comment:
actually we should not install both, but distinguish it based on the current platform (e.g. xcuitest should only be installed if the host is macOS)

Reply:
oh, right for the xcuitest